package.json

```diff
@@ -62,6 +62,8 @@
     "ps-tree": "1.2.0",
     "react-beautiful-dnd": "10.1.1",
     "react-custom-scrollbars": "4.2.1",
+    "react-intl": "^2.8.0",
```

Comment:
Nit: Please pin the version by removing the `^` for react-intl and react-intl-redux.